### DISCUSSION TITLE
Is there a way to stop test execution when one test(scenario) fails in your test file?

### DISCUSSION THREAD

**Discussion by AakashAshok** [codeceptjs/CodeceptJS#3241] [Q&A] [answered]
I currently have a test file(.ts file) in codecept with 70 scenarios, if the 5th test scenario fails - is there a way I can stop the other 65 test scenarios from running?

**Comment by jancorvus** [comment, marked as answer]
Run with the key `--bail` 
```
npx codeceptjs run --bail <the rest of params>
```
It is strange that this is absent in the documentation.

**Reply by AakashAshok** [reply]
Thanks for that - I currently run parallel execution with codecept. Lets say if I want to run three test files(.ts files) each with different yarn commands parallelly( I just want to run one test file with --bail) - is that possible?
